spyder_kernels/customize/spydercustomize.py

```diff
@@ -22,6 +22,8 @@
 import logging
 import cmd
 
+from distutils.version import LooseVersion
```

Comment:
`distutils` is part of the standard library, so it needs to go in the previous section of imports. Could you also organize them in alphabetical order as part of this PR? Thanks!